main.go

```diff
@@ -114,12 +152,14 @@ func main() {
 
 						lag := offset - partitionOffsets[partitionID]
 
-						if *useTags {
-							var tags []string
-							tags = append(tags, "topic="+topic)
-							tags = append(tags, fmt.Sprintf("partition=%d", partitionID))
-							tags = append(tags, "consumer_group="+cg)
-							stats.Gauge(fmt.Sprintf("consumer_lag,%s", strings.Join(tags, ",")), lag)
+						stats := statsdClient.Clone(
+							statsd.Tags("topic", topic),
+							statsd.Tags("partition", strconv.FormatInt(int64(partitionID), 10)),
```

Comment:
Is there any risk of this throwing? Say `partitionID` is nil?

Reply:
`partitionID` is an int32.  Int32 isn't a pointer, so it can't be nil.  This is just doing a int->str conversion.